decoupled-site/frontend/src/pages/SearchPage.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import Footer from "../components/Footer";
+import Header from "../components/Header";
+import SearchButton from "../components/SearchButton";
+import { SearchQuery, useSearchQuery } from "../generated";
+import { generateGQLSearchQueryVars } from "../helpers/queryCacheHelper";
+import { getImageUrl, isEditOrPreviewMode } from "../helpers/urlHelper";
+
+const singleKeyUrl = process.env.REACT_APP_CONTENT_GRAPH_GATEWAY_URL as string
+
+function SearchPage() {
+    const [token, setToken] = useState("")
+    const [searchParams] = useSearchParams()
+    const modeEdit = isEditOrPreviewMode()
+    let data: SearchQuery | undefined = undefined
+    let sortOption: string = "ASC"
+    let queryString: string | null
+    let resultNumber : number | undefined
+    let variables: any
+    let options: {value: string; key: string}[] = [
+        {value: "ASC", key: "ASC"},
+        {value: "DESC", key: "DESC"}
+    ]
+
+    queryString = searchParams.get("q")
+    variables = generateGQLSearchQueryVars(token, window.location.pathname, queryString, sortOption);
+
+    const { data : searchQueryData } = useSearchQuery({ endpoint: singleKeyUrl }, variables, { staleTime: 2000, enabled: !modeEdit || !!token });
+    data = searchQueryData
+    resultNumber = data?.ArtistDetailsPage?.items?.length ?? 0
+
+    return (
+        <div>
+            <Header />
```

Comment:
Where is the header? Can we keep the header on the landing page?

![Screenshot 2023-04-24 at 13 37 02](https://user-images.githubusercontent.com/1325546/233985667-26eb056a-a189-4ed2-a31c-16a5ee170af4.png)